code/datums/jobs.dm

```diff
@@ -772,32 +772,9 @@ ABSTRACT_TYPE(/datum/job/engineering)
 			return
 		M.bioHolder.AddEffect("training_miner")
 
-/datum/job/engineering/mechanic
-	name = "Mechanic"
-	limit = 4
-	wages = PAY_DOCTORATE
-
-	slot_back = list(/obj/item/storage/backpack/engineering)
-	slot_belt = list(/obj/item/storage/belt/utility/prepared)
-	slot_jump = list(/obj/item/clothing/under/rank/mechanic)
-	slot_foot = list(/obj/item/clothing/shoes/black)
-	slot_lhan = list(/obj/item/storage/toolbox/electrical/mechanic_spawn)
```

Comment:
The engineer starting equipment needs to be adjusted to make sure nothing that mechanics get goes missing in this transfer

Reply:
Done, added scanner to engineer

Reply:
Swapped the banana with a scanner